docs-core/src/main/java/com/sismics/docs/core/model/jpa/Review.java

```diff
@@ -0,0 +1,102 @@
+package com.sismics.docs.core.model.jpa;
+
+import java.util.Date;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+import com.google.common.base.MoreObjects;
+
+/**
+ * Link between an user and a group.
+ * 
+ * @author jessedin
+ */
+@Entity
+@Table(name = "T_REVIEW")
+public class Review {
```

Comment:
Shoot almost forgot this: Need to add `implements Loggable` for compatibility reasons